modules/tensor_mechanics/include/auxkernels/NewmarkVelAux.h

```diff
@@ -0,0 +1,45 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+
+#ifndef NEWMARKVELAUX_H
+#define NEWMARKVELAUX_H
+
+#include "AuxKernel.h"
+
+
```

Comment:
No double empty lines, please. (and the other empty line comments from above apply as well)